packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -368,14 +367,12 @@
                         data-term="{{:: 'hosting_dashboard_service_cdn' | translate }}"
                     >
                         <oui-tile-description>
-                            <span
-                                data-ng-if=""
-                            >
+                            <span data-ng-if="">
```

Comment:
i know it was here before, but it's useless i think.

Reply:
Thanks @mohammed-zahaf  for pointing out this useless element.
We'll handle it in a separate Pull Request.
Thanks!